### ISSUE TITLE
Details for reproducing LeNet-5 results in ICML paper

### ISSUE THREAD

**Issue by ifed-ucsd** [senya-ashukha/variational-dropout-sparsifies-dnn#11]
Can you please specify the training details for generating the LeNet5 pruning results in your paper? Did you pretrain the network or use the warm-up procedure for the KL divergence term? What learning rate did you use?

**Comment by senya-ashukha** [comment]
Hi,

We trained LeNet5 without ether pre-training or warming up KL-term. Our version of the experiment is the [file](https://github.com/ars-ashuha/variational-dropout-sparsifies-dnn/blob/master/experiments/lenet/lenet5-ard.py), to best of my memory it will reproduce the result. From our experience initialization of log_sigma2 and the schedule of a learning-rate may make a big impact.

My apologies for the late response, we had a quite busy time at the end of the year, that was followed by public holidays.
